### DISCUSSION TITLE
Cimemachine Renderering

### DISCUSSION THREAD

**Discussion by Jagglapaff** [visualdesigncafe/nature-renderer#56] [Help]
Hi. I noticed that with the game I'm currently working on that with Virtual cinemachine cameras the game will render more then needed. Some area behind the camera, extended more to the side. Is there something I need to do besides adding nature renderer to the virtualcameraBrain? or other changes?

**Comment by maxplooi** [comment]
Do you have a screenshot of this issue? There should be no additional steps necessary to use Nature Renderer with Cinemachine. Any number of cameras should work fine.
